engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41710to41800.java

```diff
@@ -124,10 +124,10 @@ protected Map<Long, String> selectValuesWithCustomConfigs(Connection conn, Strin
         }
     }
 
-    protected void executeUpdateOnConfigValues(Connection conn, Map<Long, String> valuesMapToBeUpdated, String table) {
+    protected void updateConfigValuesWithScope(Connection conn, Map<Long, String> configsToBeUpdated, String table) {
         String updateConfigValues = String.format("UPDATE cloud.%s SET value = ? WHERE id = ?;", table);
 
-        for (Map.Entry<Long, String> map : valuesMapToBeUpdated.entrySet()) {
+        for (Map.Entry<Long, String> config : valuesMapToBeUpdated.entrySet()) {
```

Comment:
changed the name from `map` to `config`, but in the code below `map` is still referenced.

Reply:
@DaanHoogland I already made these changes in the latest commit, after applying the suggestions.